### ISSUE TITLE
Outputs inconsistent for model imported to dl4j that trained by keras 

### ISSUE THREAD

**Issue by bjuthjliu** [eclipse/deeplearning4j#8047]
#### Issue Description
I trained a model by keras (v2.2.4), then loaded the model by dl4j(1.0.0-beta4).
I found that the score predicted by keras is different form dl4j.

I have checked the weights of both keras model and dl4j(after loading)，and I confirm that the weights are the same.
I've noticed https://github.com/eclipse/deeplearning4j/issues/3659, but got no insight.   @maxpumperla

The model Architecture is below:
__________________________________________________________________________________________________
Layer (type)                    Output Shape         Param #     Connected to                     
==================================================================================================
1_inp (InputLayer)    (None, 5)            0                                            
__________________________________________________________________________________________________
2_inp (InputLayer)         (None, 32)           0                                            
__________________________________________________________________________________________________
3_inp (InputLayer)        (None, 256)          0                                            
__________________________________________________________________________________________________
4_inp (InputLayer)          (None, 1)            0                                            
__________________________________________________________________________________________________
5_inp (InputLayer)     (None, 5)            0                                            
__________________________________________________________________________________________________
6_inp (InputLay (None, 4)            0                                            
__________________________________________________________________________________________________
7_inp (InputLa (None, 4)            0                                            
__________________________________________________________________________________________________
8_inp (InputLayer)    (None, 4)            0                                            
__________________________________________________________________________________________________
embedding_1 (Embedding)         (None, 5, 32)        751744      packageName_inp[0][0]            
__________________________________________________________________________________________________
embedding_2 (Embedding)         (None, 32, 64)       767040      appKey_inp[0][0]                 
__________________________________________________________________________________________________
embedding_3 (Embedding)         (None, 256, 128)     39423488    install_inp[0][0]                
__________________________________________________________________________________________________
embedding_4 (Embedding)         (None, 1, 32)        501856      model_inp[0][0]                  
__________________________________________________________________________________________________
embedding_5 (Embedding)         (None, 5, 32)        2035872     deviceName_inp[0][0]             
__________________________________________________________________________________________________
embedding_6 (Embedding)         (None, 4, 16)        2976        click_channel_ids_inp[0][0]      
__________________________________________________________________________________________________
embedding_7 (Embedding)         (None, 4, 16)        58624       click_campaign_ids_inp[0][0]     
__________________________________________________________________________________________________
embedding_8 (Embedding)         (None, 4, 16)        2352        click_goods_inp[0][0]            
__________________________________________________________________________________________________
average_pooling1d_1 (AveragePoo (None, 1, 32)        0           embedding_1[0][0]                
__________________________________________________________________________________________________
average_pooling1d_2 (AveragePoo (None, 1, 64)        0           embedding_2[0][0]                
__________________________________________________________________________________________________
average_pooling1d_3 (AveragePoo (None, 1, 128)       0           embedding_3[0][0]                
__________________________________________________________________________________________________
average_pooling1d_4 (AveragePoo (None, 1, 32)        0           embedding_4[0][0]                
__________________________________________________________________________________________________
average_pooling1d_5 (AveragePoo (None, 1, 32)        0           embedding_5[0][0]                
__________________________________________________________________________________________________
average_pooling1d_6 (AveragePoo (None, 1, 16)        0           embedding_6[0][0]                
__________________________________________________________________________________________________
average_pooling1d_7 (AveragePoo (None, 1, 16)        0           embedding_7[0][0]                
__________________________________________________________________________________________________
average_pooling1d_8 (AveragePoo (None, 1, 16)        0           embedding_8[0][0]                
__________________________________________________________________________________________________
flatten_1 (Flatten)             (None, 32)           0           average_pooling1d_1[0][0]        
__________________________________________________________________________________________________
flatten_2 (Flatten)             (None, 64)           0           average_pooling1d_2[0][0]        
__________________________________________________________________________________________________
flatten_3 (Flatten)             (None, 128)          0           average_pooling1d_3[0][0]        
__________________________________________________________________________________________________
flatten_4 (Flatten)             (None, 32)           0           average_pooling1d_4[0][0]        
__________________________________________________________________________________________________
flatten_5 (Flatten)             (None, 32)           0           average_pooling1d_5[0][0]        
__________________________________________________________________________________________________
flatten_6 (Flatten)             (None, 16)           0           average_pooling1d_6[0][0]        
__________________________________________________________________________________________________
flatten_7 (Flatten)             (None, 16)           0           average_pooling1d_7[0][0]        
__________________________________________________________________________________________________
flatten_8 (Flatten)             (None, 16)           0           average_pooling1d_8[0][0]        
__________________________________________________________________________________________________
concatenate_2 (Concatenate)     (None, 336)          0           flatten_1[0][0]                  
                                                                 flatten_2[0][0]                  
                                                                 flatten_3[0][0]                  
                                                                 flatten_4[0][0]                  
                                                                 flatten_5[0][0]                  
                                                                 flatten_6[0][0]                  
                                                                 flatten_7[0][0]                  
                                                                 flatten_8[0][0]                  
__________________________________________________________________________________________________
dense_4 (Dense)                 (None, 64)           21568       concatenate_2[0][0]              
__________________________________________________________________________________________________
batch_normalization_3 (BatchNor (None, 64)           256         dense_4[0][0]                    
__________________________________________________________________________________________________
activation_3 (Activation)       (None, 64)           0           batch_normalization_3[0][0]      
__________________________________________________________________________________________________
dropout_3 (Dropout)             (None, 64)           0           activation_3[0][0]               
__________________________________________________________________________________________________
dense_5 (Dense)                 (None, 32)           2080        dropout_3[0][0]                  
__________________________________________________________________________________________________
batch_normalization_4 (BatchNor (None, 32)           128         dense_5[0][0]                    
__________________________________________________________________________________________________
activation_4 (Activation)       (None, 32)           0           batch_normalization_4[0][0]      
__________________________________________________________________________________________________
dropout_4 (Dropout)             (None, 32)           0           activation_4[0][0]               
__________________________________________________________________________________________________
dense_6 (Dense)                 (None, 1)            33          dropout_4[0][0]                  
==================================================================================================
Total params: 43,568,017
Trainable params: 43,567,825
Non-trainable params: 192
__________________________________________________________________________________________________

#### Version Information
* Deeplearning4j 1.0.0-beta4
* keras 2.2.4
